easybuild/easyconfigs/w/WRF/WRF-3.6.1-foss-2015a-dmpar.eb

```diff
@@ -0,0 +1,40 @@
+name = 'WRF'
+version = '3.6.1'
+
+homepage = 'http://www.wrf-model.org'
+description = """The Weather Research and Forecasting (WRF) Model is a next-generation mesoscale
+ numerical weather prediction system designed to serve both operational forecasting and atmospheric
+ research needs."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+toolchainopts = {'opt': False}  # don't use agressive optimization, stick to -O2
```

Comment:
No, `opt` (which corresponds to `-O3`) is the default. Disabling `opt` implicitly makes `defaultopt` the active one, which corresponds to `-O2`.

See https://github.com/hpcugent/easybuild-framework/blob/master/easybuild/tools/toolchain/compiler.py#L101

Yes, we need better documentation for this, or a dedicated cmdline option to query this.